system/Debug/Toolbar/Collectors/Routes.php

```diff
@@ -88,7 +88,7 @@ public function display(): array
 		$route = $router->getMatchedRoute();
 
 		// Get our parameters
-		$method    = is_callable($router->controllerName()) ? new \ReflectionFunction($router->controllerName()) : new \ReflectionMethod($router->controllerName(), $router->methodName());
+		$method    = is_callable($router->controllerName()) ? new \ReflectionFunction($router->controllerName()) : new \ReflectionMethod($router->controllerName(), method_exists($router->controllerName(), '_remap') ? '_remap' : $router->methodName());
```

Comment:
I guess this will always go for remap...

Reply:
nope it's not always go for remap. only when controller contains remap method. the main issue is, when controller contains remap method only, it creates error. $router->methodName() always return "index" when controller contains remap method only. if controller having both remap and index method, toolbar always choose index instead of remap. but router always prefers remap first. calling ReflectionMethod without checking method exists check creates ReflextionException error.

Reply:
What I am saying here is,
for example, method abc() is present and I am calling same method from route, then also, according to your condition, if both method _remap() and abc() are present, it will always go for _remap().
But ideally, it should go directly to abc()

Reply:
yup, you are right. but if you have controller with only one method, that is remap. then router correctly choose remap method, but toolbar always fallback to default method **index**, without checking method existence.